### ISSUE TITLE
Fix incorrect timestamps in test data/examples

### ISSUE THREAD

**Issue by clenk** [oasis-open/cti-python-stix2#230]
`created` and `modified` timestamps must be to the nearest millisecond, but in some of our test data and examples they have greater precision. Example: https://github.com/oasis-open/cti-python-stix2/blob/v1.1.0/stix2/test/v21/stix2_data/course-of-action/course-of-action--95ddb356-7ba0-4bd9-a889-247262b8946f/20170531213026495974.json We need to fix the examples and make sure there aren't any bugs in our DataStore code that saves objects to the filesystem.

Related: #24, #223.

**Comment by chisholm** [comment]
Do you think we should be lenient on what we accept as input, but more strict on output?  If so, we need some test data with more precision, for use as input, to test with.

**Comment by clenk** [comment]
Yes, I agree. I think test input that varies from the spec should be in the Python test cases, though, while the files in stix2_data should be spec-compliant unless clearly labeled.
